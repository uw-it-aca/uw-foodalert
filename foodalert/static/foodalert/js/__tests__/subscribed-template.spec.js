"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import SubscribedController from '../components/subscribed-controller.vue';

describe('Subscribed Controller', function () {
        test('renders correctly', function () {
            var wrapper = mount(SubscribedController, {
            });
            expect(wrapper.element).toMatchSnapshot();
        });
})
