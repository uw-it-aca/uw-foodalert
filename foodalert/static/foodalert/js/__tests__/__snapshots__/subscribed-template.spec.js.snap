// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscribed Controller renders correctly 1`] = `
<div
  class="container"
>
  <p>
    
        Congratulations! You are signed up to be notified about leftover food after on-campus events.
    
  </p>
   
  <p>
    
        You will be emailed at
    <br />
     
    <strong>
       example@email.org 
    </strong>
  </p>
   
  <a
    class="btn btn-info w-100 my-4 py-2"
    href="/signup"
    target="_self"
  >
    <span
      class="ml-2"
    >
      Edit Notification Settings
    </span>
  </a>
   
  <button
    class="btn w-100 py-2 btn-outline-secondary"
    type="button"
  >
    
        Unsubscribe
    
  </button>
</div>
`;
